src/Symfony/Bundle/SecurityBundle/Templating/Helper/LogoutUrlHelper.php

```diff
@@ -25,33 +23,15 @@
 class LogoutUrlHelper extends Helper
 {
     private $generator;
-    private $listeners = array();
-    private $router;
-    private $tokenStorage;
```

Comment:
if these are already unused in the current code, please submit a PR removing them in the 2.7 branch to clean the code